CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 3.6.0 (YYYY-MM-DD)
 
+### Breaking Changes
+
+* [ObjectServer] Read-only is not supported by synced Realm. "SyncConfigration.readOnly()" will always throw `UnsupportedOperationException`.
```

Comment:
Do other bindings have the same realm use cases? If yes, we should just change the behaviour in Object Store.  relevant discussion on slack https://realmio.slack.com/archives/C0340J37W/p1494485682035749 https://realmio.slack.com/archives/C053279FS/p1494422645218828